err.go

```diff
@@ -598,7 +598,12 @@ func New(msg string) *Err {
 	return toErr(nil, msg, nil)
 }
 
-// Wrap returns a  clues.Err with a new message wrapping the old error.
+// NewWith is equivalent to clues.New("msg").WithClues(ctx)
+func NewWith(ctx context.Context, msg string) *Err {
```

Comment:
I'm meh on the naming.  Thoughts for alternatives?  `NewCtx`?  `NewC`?

Reply:
heh WTB function overloading based on parameters. Besides that though, I'd honestly say `NewWithClues` would be the most clear in this case even though it's a bit wordier

Reply:
Aye, that'd be a great feature.  I think `FooWithClues` is a bit too wordy for my tastes.  The more common and boilerplate a func, the more reasonable it is to be less wordy and more concise, since maintainer familiarity will catch on quick.